app/views/shared/steps/_form_notice.html.erb

```diff
@@ -4,7 +4,7 @@
 %>
 <% if message %>
   <div class="row">
-    <div class="col-md-7">
+    <div class="col-md-12">
```

Comment:
How is this change related to the change to tabbed navigation?

Reply:
the tabs now span over 12 columns, therefore, the error message should follow a consistent width

Reply:
https://github.com/alphagov/collections-publisher/pull/438/commits/28c55b952e4ba32934bc99f88360378efa4508a1